src/components/AppHeader.vue

```diff
@@ -1,55 +1,23 @@
 <template>
-  <header class="header__menu">
+  <header class="header">
     <div class="container">
-      <div class="header__navbar">
-        <div class="header__menu--left-side">
-          <img
-            class="header__menu--logo"
-            src="@/assets/img/main_logo.svg"
-            alt="Food delivery website logo"
-          />
-
-          <div class="header__menu--search">
-            <input
-              class="header__menu--search-input"
-              type="text"
-              placeholder="Search "
-            />
-            <i
-              class="header__menu--search-icon fa-solid fa-magnifying-glass"
-            ></i>
+      <div class="header-wrapper">
+        <div class="header-left">
+          <img class="logo" src="@/assets/header/main_logo.svg">
+          <div class="search-form">
+            <input class="search-input" type="text" placeholder="Search ">
+            <i class="search-icon fa-solid fa-magnifying-glass"></i>
           </div>
         </div>
-
-        <div class="header__menu--right-side">
-          <nav class="header__menu--link">
-            <router-link class="menu_link" to="">Restaurants</router-link>
-            <a class="menu_link" href="#">Deals</a>
-            <span class="header__menu--divider-vertical"></span>
-            <a class="menu_link active" href="#">My orders</a>
-          </nav>
-          <div class="header__menu--icon">
-            <a href="#"
-              ><img
-                class="header__menu--icon-item"
-                src="@/assets/img/shopping bag.svg"
-                alt="shopping bag icon"
-            /></a>
-            <span class="header__menu--icon-number">4</span>
-          </div>
-          <router-link to="/profile">
-            <img
-              class="header__menu--useravatar"
-              src="@/assets/img/useravatar.svg"
-              alt="useravatar img"
-            />
-          </router-link>
-          <span class="header__menu--divider-vertical"></span>
-          <img
-            class="header__menu--icon-mobile"
-            src="@/assets/img/menu.svg"
-            alt=" header menu icon "
-          />
+        <div class="menu">
+          <router-link class="menu__link" to="#">Restaurants</router-link>
```

Comment:
We don't have these routes yet, please remove them

Reply:
@nuronbeck Fixed!